src/shared/hooks/api-client.hook.ts

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+import { apiClient } from '../libs/api-client.lib';
+import { useAuth } from './auth.hook';
+
+export function useApiClient() {
```

Comment:
Why?

Reply:
> Why?

Because I want to have one different clients for without auth and with auth requests? Should I do it differently

Reply:
> > Why?
> 
> Because I want to have one different clients for without auth and with auth requests? Should I do it differently

Fixed